### PR TITLE
adding new queries

### PR DESCRIPTION
**Pull request by marissaposner**
Adding examples for GraphQL schema

### CODE REVIEW

File: backend/backend/services/openAI/prompts/compound_governance.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+Prompt(
+q="How many votes did proposal 86 have?",
+o="""query{
+  proposals(where: {id: "86"}) {
+    abstainWeightedVotes
+    againstWeightedVotes
+    forWeightedVotes
+  }
+}"""),
```
Comment (gosuto-inzasheru) [inline backend/backend/services/openAI/prompts/compound_governance.py:13]: shouldnt these technically be summed? there is also a `totalWeightedVotes`.

i would opt for:
- how many votes did prop x have? -> totalweightedvotes
- what were the results for prop x? -> abstain/against/for

File: backend/backend/services/openAI/prompts/compound_governance.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    blockNumber
+    abstainWeightedVotes
+    againstWeightedVotes
+    forWeightedVotes
+    timestamp
+    proposal {
+      id
+    }
+  }
+}"""),
```
Comment (gosuto-inzasheru) [inline backend/backend/services/openAI/prompts/compound_governance.py]: query does not answer the question imo?

File: backend/backend/services/openAI/prompts/compound_governance.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+Prompt(
+q="Who are the top delegates by voting power?",
+o="""query{
+  votes(orderBy: voter__tokenHoldersRepresentedAmount, orderDirection: desc) {
+    voter {
+      tokenHoldersRepresentedAmount
```
Comment (gosuto-inzasheru) [inline backend/backend/services/openAI/prompts/compound_governance.py:73]: i think you need ~~`delegatedAmount`~~`delegatedVotes` instead; `tokenHoldersRepresentedAmount` counts the amount of delegators but not the number of votes they delegated.

File: backend/backend/services/openAI/prompts/compound_governance.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      id
+    }}}
+    """),
+Prompt(
+q="What is the total number of proposals, number of proposals queued for execution, number of proposals executed, and number of proposals canceled for a governance with the ID 85?",
```
Comment (gosuto-inzasheru) [inline backend/backend/services/openAI/prompts/compound_governance.py:78]: i dont get this question. what is a governance with id 85?

afaik proposals have ids. do you mean to ask for the total number of votes instead?

File: backend/backend/services/openAI/prompts/compound_governance.py (hunk 1 above)
Comment (marissaposner) [reply backend/backend/services/openAI/prompts/compound_governance.py:13]: i agree

File: backend/backend/services/openAI/prompts/compound_governance.py [hunk 5]
```diff
@@ -2,16 +2,23 @@
...
     forWeightedVotes
   }
 }"""),
+Prompt(
+q="How many votes did proposal 86 have?",
+o="""query{
+  proposals(where: {id: "86"}) {
+    totalWeightedVotes
+  }
+}""")
```
Comment (gosuto-inzasheru) [inline backend/backend/services/openAI/prompts/compound_governance.py:20]: think this needs a comma at the end

File: backend/backend/services/openAI/prompts/compound_governance.py (hunk 4 above)
Comment (gosuto-inzasheru) [reply backend/backend/services/openAI/prompts/compound_governance.py:78]: i dont see a solution to this in the current pr

**Review by gosuto-inzasheru — COMMENTED** [review]
do we need `uniswap_governance.py`? looks like a copypaste of (old) `compound_governance.py` to me?

**Pull request closed without merging**